docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
       - "4001:4001/udp"
       - "0.0.0.0:5001:5001"
       - "0.0.0.0:8080:8080"
+    env_file:
+      - ${IPFS_ENV_FILE:-/dev/null}
```

Comment:
dockerfile syntax of optional env files doesn't work well, so this is a fix for ignoring env file if IPFS_ENV_FILE is not set. see https://github.com/docker/compose/issues/3560#issuecomment-1851794970 for more info

This allows me to set .env files for testing helia-dr(`.env.helia-dr`), helia(`.env.helia`), and helia-tg(`.env.helia-tg`) and easily change them with the following config:

```properties
# .envrc (for direnv)
dotenv .env.local
dotenv .env

# .env.local (already in this repo
...

# .env
IPFS_ENV_FILE=.env.helia-dr
IPFS_IMAGE=helia-http-gateway:local-arm64
```

and all i have to do is update .env and run `direnv reload` then `docker compose up -d` to get updated services.

![image](https://github.com/probe-lab/tiros/assets/1173416/209e5a04-3714-4cb7-8a55-adae2a41ded6)

Reply:
neat!